### PR TITLE
Updated Editor

### LINKED ISSUE #219: Edit Feature for the Editor
**Is your feature request related to a problem? Please describe.**
Closes #198

**Describe the solution you'd like**
Feature to Edit Files withe the Editor App

Can anyone assign me to this Issue?

### PR DESCRIPTION
**Pull request by Tert0**
**Description**
Added an "Edit" Feature to the Texteditor.
Added a deleted File Popup and changed File Popup to the Texteditor.

**Issue**
Closes #219 

**Testing Instructions**
- Open a file and Write something.
- Open a new Editor an open the same File.
- Save the Changes of the first Editor
- In the 2nd Editor should be a Popup.
- Now remove the File.
- Now should be an other Popup in both Editors

**Important**
You can destroy Wallets by Edit them!

### CODE REVIEW

File: src/app/desktop/windows/editor/editor.component.ts [hunk 1]
```diff
@@ -61,8 +107,37 @@ export class EditorComponent extends WindowComponent implements OnInit {
...
+    if (this.fileContent !== file.content) {
+      this.fileContent = file.content;
+    }
+  });
+  this.changed_popup = false;
+}
+
+}
+
+
```
Comment (MarcelCoding) [inline src/app/desktop/windows/editor/editor.component.ts:140]: Please correct intentions

**Review by MarcelCoding — CHANGES_REQUESTED** [review]

File: src/app/desktop/windows/editor/editor.component.ts [hunk 2]
```diff
@@ -61,8 +107,37 @@ export class EditorComponent extends WindowComponent implements OnInit {
...
+      if (this.fileContent !== file.content) {
+        this.fileContent = file.content;
+      }
+    });
+    this.changed_popup = false;
+  }
+
 }
 
+
```
Comment (JannikEmmerich) [inline src/app/desktop/windows/editor/editor.component.ts:140]: Remove unnecessary line breaks.

**Review by JannikEmmerich — CHANGES_REQUESTED** [review]

**Review by MarcelCoding — APPROVED** [review]

File: src/app/desktop/windows/editor/editor.component.scss [hunk 3]
```diff
@@ -9,36 +9,98 @@
...
   }
 
   .editor-text-field {
-    padding: 1em;
     font-family: Consolas, monospace;
+    width: 100%;
+    height: 100%;
+    display: block;
+    margin: 0px;
+    padding: 0px;
```
Comment (JannikEmmerich) [inline src/app/desktop/windows/editor/editor.component.scss:63]: Add some padding so it looks better.
```suggestion
    padding: 5px;
```

File: src/app/desktop/windows/editor/editor.component.scss [hunk 4]
```diff
@@ -9,36 +9,98 @@
...
+    box-sizing: border-box;
+    resize: none;
   }
```
Comment (JannikEmmerich) [inline src/app/desktop/windows/editor/editor.component.scss:66]: Remove the outline of the text field.
```suggestion
    outline: none;
  }
```

File: src/app/desktop/windows/editor/editor.component.ts [hunk 5]
```diff
@@ -61,8 +107,34 @@ export class EditorComponent extends WindowComponent implements OnInit {
...
+    this.fileService.getFile(this.delegate.device.uuid, this.fileUUID).subscribe(file => {
+      if (this.fileContent !== file.content) {
+        this.fileContent = file.content;
+      }
+    });
+    this.changed_popup = false;
+  }
 }
 
+
```
Comment (JannikEmmerich) [inline src/app/desktop/windows/editor/editor.component.ts:137]: Remove empty line.
```suggestion

```

**Review by JannikEmmerich — CHANGES_REQUESTED** [review]

**Review by MarcelCoding — APPROVED** [review]

**Review by JannikEmmerich — APPROVED** [review]

**Pull request merged**